general/server/plugins/index.md

```diff
@@ -45,17 +45,55 @@ All plugins hosted on the repository can be built from source and manually added
 
 ### Official Plugins
 
-#### Anime
+#### Anime Plugins
```

Comment:
I don't think we need a special category just for anime plugins. I wouldn't mind it, but it needlessly indents all the plugins one header lower.

Reply:
i had set it uo this way because kitsu is the oddball out with its name. i could format the page to match the webui, making categories for each type. or just make it normal.

Reply:
Why is Kitsu the oddball? The plugins have descriptions right below the titles to explain their purpose, so I'm not sure a category is needed.

Reply:
If anything, it would make sense to group them by category, so Metadata plugins rather than Anime plugins.
With that said, it shouldn't be too much work to whip up a dynamic page that pulls the info from the plugin repo itself, replacing this whole section.